roles/deploy_jhub/templates/config.yaml.j2

```diff
@@ -3,16 +3,16 @@ proxy:
   https:
     enabled: true
     hosts:
-      - <Host Domain>
+      - "{{ host_domain }}"
     letsencrypt:
-      # Uncomment acmeserver for a test certificate
-      # acmeServer: "https://acme-staging-v02.api.letsencrypt.org/directory"
+{% if acme_server is true %}
```

Comment:
Nit: This should be `staging_cert` as you're basically saying

`acme_server == True  # Gets a staging cert`

Reply:
We should the enter an else with the non-staging cert, since running this with `acme_server` true then running it with it set false won't reset the value, it will leave it set to: `"https://acme-staging-v02.api.letsencrypt.org/directory"`